--- talloc-2.1.6/third_party/waf/wafadmin/Options.py.orig	2016-03-12 00:03:05.936773996 +0000
+++ talloc-2.1.6/third_party/waf/wafadmin/Options.py	2016-03-12 00:03:16.769773802 +0000
@@ -36,7 +36,7 @@
 		d = tempfile.gettempdir()
 		default_prefix = d[0].upper() + d[1:]
 		# win32 preserves the case, but gettempdir does not
-	else: default_prefix = '/usr/local/'
+	else: default_prefix = '/usr/'
 
 default_jobs = os.environ.get('JOBS', -1)
 if default_jobs < 1:
