--- talloc-2.1.6/configure.orig	2016-03-12 00:27:01.071748296 +0000
+++ talloc-2.1.6/configure	2016-03-12 00:27:41.744747568 +0000
@@ -11,11 +11,14 @@
 	exit 1
 fi
 
+WAFOPTS="$@"
+WAFOPTS=`echo $WAFOPTS | sed 's/ --disable-static//'`
+
 # using JOBS=1 gives maximum compatibility with
 # systems like AIX which have broken threading in python
 JOBS=1
 export JOBS
 
 cd . || exit 1
-$WAF configure "$@" || exit 1
+$WAF configure "$WAFOPTS" || exit 1
 cd $PREVPATH
